fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -105,9 +108,13 @@ def signed_weights(self, lambda_vec):
         :param lambda_vec: The vector of Lagrange multipliers indexed by `index`
         :type lambda_vec: :class:`pandas:pandas.Series`
         """
-        lambda_signed = lambda_vec["+"] - lambda_vec["-"]
-        adjust = lambda_signed.sum(level=_EVENT) / self.prob_event \
-            - lambda_signed / self.prob_group_event
+        adjust = self.multiplier[_MULTIPLIER].mul(((lambda_vec["+"] -
```

Comment:
Is there any way this can be broken apart a bit? The previous version was already painfully long.

Reply:
Sure, I'll format it to make it more readable.

Reply:
Looks much better than before actually. We usually aim at single lines for math to make it as readable as possible, even if that means that we have to assign more variables.